docs/reference/readme.md

```diff
@@ -24,12 +24,12 @@ Pomerium can hot-reload route configuration details, authorization policy, certi
 
 :::
 
-
 ## Shared Settings
-These configuration variables are shared by all services, in all service modes.
 
+These configuration variables are shared by all services, in all service modes.
```

Comment:
Can you turn off your editors auto-formatter to isolate changes?

Reply:
Also, if there are changes to reference.md, they need to be made in `settings.yaml` then generate this file.